examples/kubeflow-jupyter-notebook/README.md

```diff
@@ -0,0 +1,66 @@
+# Using `Fairing` with Kubeflow's TfJob inside a Jupyter Notebook
+
+## Requirements
+
+### Kubeflow
+
+Kubeflow needs to be pre-installed in the cluster.
+
+### Knative Build Component
+A Knative Build controller needs to be deployed in the cluster beforehand to handle in-cluster builds.  
+See: [Installing Knative Build Component](https://github.com/knative/docs/blob/master/build/installing-build-component.md)
+
+
+### Container Registry Secret
```

Comment:
I think that we should reuse the user service account that gets created instead

Reply:
I'm not sure I understand with service account you are referring to?